resources/views/frontend/needs/add.blade.php

```diff
@@ -34,6 +34,21 @@
                             <textarea class="form-control" rows="4" id="address" name="address" required>{{ old('address') }}</textarea>
                         </div>
                     </div>
+                     <div class="form-group">
+                        <label for="latlon" class="col-lg-2 control-label">ඔබ සිටින ස්ථනය සිතියමෙන්</label>
+                        <div class="col-lg-10">
+                            <input type="hidden" class="form-control" id="latlon" name="latlon"  value="{{ old('latlon') }}"/>
+                              <button type="button" onclick="getLocation()" class="btn btn-primary">සිතියමට ඇතුල් කරන්න</button>
+                            <p id="demo">&nbsp;</p>
+                                <div id="mapholder" style="width:55%;height:300px">
+                                </div>
```

Comment:
Yes. Looks like there are several issues with different scopes on Maps. @danishka does this cover what you had in mind for #50 ?

Also #6 also seems to be similar in scope to this.